packages/grid/x-grid/src/tests/selection.XGrid.test.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { XGrid, useApiRef, ApiRef } from '@material-ui/x-grid';
+// @ts-expect-error need to migrate helpers to TypeScript
+import { fireEvent, createClientRenderStrictMode } from 'test/utils';
+import { expect } from 'chai';
+import { getCell, getColumnHeaders, getRow } from '../../../../../test/utils/helperFn';
```

Comment:
Makes moving code easier

```suggestion
import { getCell, getColumnHeaders, getRow } from 'test/utils/helperFn';
```